databroker/src/broker.rs

```diff
@@ -155,7 +155,7 @@ pub struct QuerySubscription {
 
 pub struct ChangeSubscription {
     entries: HashMap<i32, HashSet<Field>>,
-    sender: mpsc::Sender<EntryUpdates>,
+    sender: Option<mpsc::Sender<EntryUpdates>>,
```

Comment:
I'm not convinced it makes sense to convert this into an `Option<>`.. or put another way, what's the point of having a `ChangeSubscription` at all if `sender` is `None`?

Given that the cleanup will now remove subscriptions if the permission has expired, I think this (introduction of `Option<>`) should be reverted.

Reply:
> I think this (introduction of Option<>) should be reverted.

Is there any reason to have this `Option<>`? Otherwise, I still think it should be removed / reverted to how it was before this PR.